### PR TITLE
RavenDB-12874: CanPreformSeveralClusterTransactions - add NoTimeouts(…

### PR DESCRIPTION
**Pull request by aviviadi**
…) to prevent flackiness due to slow response while slow testing

### CODE REVIEW

File: test/SlowTests/Cluster/ClusterTransactionTests.cs [hunk 1]
```diff
@@ -109,6 +109,7 @@ private string RandomString(int length)
         [InlineData(5)]
         public async Task CanPreformSeveralClusterTransactions(int numberOfNodes)
         {
+            NoTimeouts();
```
Comment (karmeli87) [inline test/SlowTests/Cluster/ClusterTransactionTests.cs:112]: Can we do that? I think we are reusing the servers, no?

Comment (aviviadi) [reply test/SlowTests/Cluster/ClusterTransactionTests.cs:112]: so? are we relying on timeouts in the tests?

Comment (aviviadi) [reply test/SlowTests/Cluster/ClusterTransactionTests.cs:112]: I can set no timeouts only for the server in hand, and restore the timeout once the test is over

File: test/SlowTests/Cluster/ClusterTransactionTests.cs [hunk 2]
```diff
@@ -109,70 +109,80 @@ private string RandomString(int length)
         [InlineData(5)]
         public async Task CanPreformSeveralClusterTransactions(int numberOfNodes)
         {
```
Comment (ayende) [inline test/SlowTests/Cluster/ClusterTransactionTests.cs:111]: We should do this only on our server, so we need `UseLocalServer` here too, I think

Comment (aviviadi) [reply test/SlowTests/Cluster/ClusterTransactionTests.cs:111]: Not needed: `var leader = await CreateRaftClusterAndGetLeader(numberOfNodes);` is being used in this test (creates new servers)

**Pull request merged**